lib/skate_web/channels/alerts_channel.ex

```diff
@@ -7,13 +7,18 @@ defmodule SkateWeb.AlertsChannel do
 
   @impl SkateWeb.AuthenticatedChannel
   def join_authenticated("alerts:route:" <> route_id, _message, socket) do
-    alerts = Duration.log_duration(Server, :subscribe_to_alerts, [route_id])
-    {:ok, %{data: alerts}, socket}
+    {lookup_key, alerts} = Duration.log_duration(Server, :subscribe_to_alerts, [route_id])
+
+    {:ok, %{data: alerts},
+     Phoenix.Socket.assign(socket, Map.merge(socket.assigns, %{lookup_key: lookup_key}))}
```

Comment:
question: is this merge required? The docs seem to suggest that Socket.assign/2 & /3 already merge the provided keys to the assigns map.
https://hexdocs.pm/phoenix/1.6.16/Phoenix.Socket.html#assign/2

Reply:
```suggestion
     Phoenix.Socket.assign(socket, lookup_key: lookup_key)
```

Reply:
- https://github.com/mbta/skate/blob/1478499366e9eb5e65cc0f51da6c67bec8aec83d/lib/skate_web/channels/vehicle_channel.ex#L28
- https://github.com/mbta/skate/blob/1478499366e9eb5e65cc0f51da6c67bec8aec83d/lib/skate_web/channels/vehicle_channel.ex#L63
- https://github.com/mbta/skate/blob/1478499366e9eb5e65cc0f51da6c67bec8aec83d/lib/skate_web/channels/vehicles_channel.ex#L14-L49
- https://github.com/mbta/skate/blob/1478499366e9eb5e65cc0f51da6c67bec8aec83d/lib/skate_web/channels/vehicles_channel.ex#L97
- https://github.com/mbta/skate/blob/1478499366e9eb5e65cc0f51da6c67bec8aec83d/lib/skate_web/channels/vehicles_search_channel.ex#L49
- https://github.com/mbta/skate/blob/1478499366e9eb5e65cc0f51da6c67bec8aec83d/lib/skate_web/channels/vehicles_search_channel.ex#L87

Reply:
@firestack I actually just force-pushed a change to fix all of those other spots, there should be no more uses of `Map.merge` in any of the calls to `Socket.assign`.

Reply:
Thanks!